src/links/links.service.ts

```diff
@@ -1,8 +1,13 @@
-import { Injectable, NotFoundException } from '@nestjs/common';
-import { Link, Prisma } from '@prisma/client';
+import {
+  ConflictException,
```

Comment:
**suggestion**: Consider organizing imports alphabetically for better readability.